### PR TITLE
fix: badge added to color picker in product card

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

Closes #

# Scope of work
Add space between image and color picker on Product Card to avoid overlapping when there are many colors to chose. 
Screenshot from 2021-11-24 15-42-25

# Screenshots of visual changes

# Checklist

- [ ] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Comment by aniamusial** [comment]
@AdamPawlinski please update the changelog :) This should go to 11.4

**Review by aniamusial — APPROVED** [review]

**Review by aniamusial — APPROVED** [review]

**Review by justyna-13 — APPROVED** [review]
Looks good on desktop :) and what do you think about adding white color on a font in a badge on mobile? 
Screenshot from 2021-12-08 08-41-53

File: packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue [hunk 1]
```diff
@@ -311,6 +321,10 @@ export default {
       const defaultClass = "sf-button--pure sf-product-card__wishlist-icon";
       return `${defaultClass} ${this.isInWishlist ? "on-wishlist" : ""}`;
     },
+    showBadge() {
+      console.log(this.colors.length);
```
Comment (aniamusial) [inline packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue:325]: chore: remove console log :)

**Review by aniamusial — APPROVED** [review]

**Comment by AdamPawlinski** [comment]
Color fixed on mobile.

**Pull request merged**